### DISCUSSION TITLE
新加坡联合早报：中国政府紧急约见银行讨论烂尾楼停贷潮问题

### DISCUSSION THREAD

**Discussion by WeNeedHome** [WeNeedHome/SummaryOfLoanSuspension#366] [General]
https://www.zaobao.com/realtime/china/story20220714-1292831

![image](https://user-images.githubusercontent.com/109166931/178940027-8ec8dd1f-a3e6-43a6-ba3b-2b62cbd037d3.png)

**Comment by WeNeedHome** [comment]
刚出的新闻，联合早报有一个专门板块报道此事了。

**Comment by douloveppt** [comment]
好耶👍🏻

**Comment by radiateboy** [comment]
静待佳音

**Comment by BuryMe** [comment]
看来快压不住了，地方终于捅到中央了。

**Reply by WallStreetHu** [reply]
应该捅到他们嗓子眼😂

**Reply by SakuraMeii** [reply]
不是ass?

**Comment by Jesse3692** [comment]
只要我黄老爷在，鹅城就翻不了天

**Reply by DarkForestBlack** [reply]
黄老爷的身份可以深入说一说。^_^

**Reply by ruo-xi** [reply]
> 黄老爷的身份可以深入说一说。^_^

我猜肯定有  邓家  朱镕基？

**Comment by ghost** [comment]


**Reply by gmars** [reply]
想多了。印钞了怎么能在国际上证明我们经济如雨后春笋？网监系统加几个屏蔽关键词就能解决的问题。

**Comment by AlanTianx** [comment]
尔俸尔禄,民膏民脂,下民易虐,上天难欺

**Reply by guanyuwanglei** [reply]
盛世中华

**Comment by AlanTianx** [comment]
_尔俸尔禄,民膏民脂,下民易虐,上天难欺_

**Comment by haha110110** [comment]
然后各大银行宣布影响不大

**Comment by generals-space** [comment]
原来中央真的不知道啊?

**Reply by peng1987123** [reply]
肯定是知道的，国务院分管的头头肯定是知道的。

**Reply by generals-space** [reply]
只是假装不知道而已, 如果真的不知道的话, 那就证明他们已经离开人民太远太久了

**Comment by xclidongbo** [comment]
总体风险可控。这句话，已经把这三百多个楼盘的业主。定性了。要么还贷款，要么当老赖。
